fc-community-backend/apps/api/src/factchain-core/web3.ts

```diff
@@ -74,37 +69,35 @@ export class FactChainBackend implements NoteReader, NoteWriter {
     };
   };
 
-  getNotes = async (postUrl: string): Promise<Array<Note>> => {
+  getNotes = async (
+    predicate: (note: Note) => boolean,
```

Comment:
can you rename `predicate` to something more meaningful? Even something basic like `filterFn`.

Reply:
![Screenshot 2024-01-04 at 16 00 20](https://github.com/factchain/factchain-community/assets/15421103/78617d10-7d5c-443f-a518-712ba6fcb70d)


predicate is the conventional naming for higher order function given to filter.